### ISSUE TITLE
Filtering: unfiltered spambot

### ISSUE THREAD

**Issue by agardip** [ThioJoe/YT-Spammer-Purge#671]
### Filter Mode

Auto-Smart Mode

### Select the Problem

A type of spammer is not detected at all

### Spammer Example / Sample

username: [T[A]P Me!! To Have [S]EX With Me]
channel: https://www.youtube.com/channel/UCstn5_sHhGMUM-B16D6G8vQ

[Spam_Log_2022-02-17_19-00-33.txt](https://github.com/ThioJoe/YT-Spammer-Purge/files/8091137/Spam_Log_2022-02-17_19-00-33.txt)

### Video / Post Link

https://www.youtube.com/watch?v=k_OOiuaZSKc&lc=Ugyqo23wVgMpw6RBXF54AaABAg

### (Optional) Additional Info / Context

simple "copy a comment" type of spambot.

Thank you for your effort to make YouTube a better place!

**Comment by Firecul** [comment]
What version are you using? It's picked up for me.

![WindowsTerminal_Windows_PowerShell (32)](https://user-images.githubusercontent.com/34950371/154546967-90af37a8-d7ea-4f9e-9d50-f13a86e5ce3a.png)

**Comment by ThioJoe** [comment]
Hm so your program didn't alert you there's a new version while in the main menu?

**Comment by agardip** [comment]
> Hm so your program didn't alert you there's a new version while in the main menu?

I probably did, because I just started the old version and it's there.

Well, I used the program a couple of times so I became a bit automated. But if I'm not automated I'm really looking at the numbered options, which are located on the far left, while the upgrade info (especially the "Yes") if on the far right. Because of my color seeing deficiency I learned to not rely on them and even if it were good the main menu is very colorful, hence the little "Yes" on the far right did not caught my attention.
